README.md

```diff
@@ -24,8 +24,8 @@ If you have found an issue with the application, please [file a bug](https://bug
 
 ## Get in touch
 
-We're friendly! Talk to us on [IRC](https://webchat.freenode.net/?channels=snappy)
```

Comment:
I was going to put Rocket Chat first, then checked the Rocket Chat activity after noticing I'd got the channel link wrong and decided to put IRC first :P I'm guessing you'd still like to use Rocket Chat as an active development channel so I've left it in?